Mattvol4 (#118)
rapid7/insightconnect-plugins: duo_admin/help.md

# Description

[Duo](https://duo.com/)'s Trusted Access platform verifies the identity of your users with two-factor authentication and

> security health of their devices before they connect to the apps they use. The InsightConnect plugin can be used to manage Duo users.

Reviewer: ```suggestion
security health of their devices before they connect to the apps they use. Using the Duo plugin for InsightConnect will allow Duo user management within automation workflows.
```

# Key Features

> * Add and Delete users

Reviewer: ```suggestion
* Add and delete users
```

* Enroll users

> * Get User information

Reviewer: ```suggestion
* Get user information
```
